### ISSUE TITLE
Deprecation warning for type_aliases

### ISSUE THREAD

**Issue by brettc** [ion-elgreco/polars-hash#27]
Hi. Thanks for making this nice library. Not pressing, but it looks like type_aliases have been deprecated in polars 1.3.0.

```
DeprecationWarning: The `polars.type_aliases` module is deprecated. The type aliases have moved to the `polars._typing` module to explicitly mark them as private. Please define your own type aliases, or temporarily import from the `polars._typing` module. A public `polars.typing` module will be added in the future.
  import polars_hash as plh
```

**Comment by ion-elgreco** [comment]
@brettc yes I got aware of that, I'm waiting on next release of Polars and pyo3-polars so I can do a single release

**Comment by Filimoa** [comment]
Seconding

**Comment by ion-elgreco** [comment]
I'm extremely busy atm so would welcome a PR to address this

**Comment by Filimoa** [comment]
#28

**Comment by Filimoa** [comment]
Merged in `0.4.9` - I think we're good to close this issue!
